cdc/owner.go

```diff
@@ -750,6 +754,9 @@ func (o *Owner) flushChangeFeedInfos(ctx context.Context) error {
 		}
 	}
 	if time.Since(o.gcSafepointLastUpdate) > GCSafepointUpdateInterval {
+		if (time.Now().Unix() - int64(minCheckpointTs)) > o.gcBLT {
+			minCheckpointTs = uint64(time.Now().Unix() - o.gcBLT)
```

Comment:
`time.Now()` could have clock drift, it is not safe to operate `time.Now().Unix()` with a tso. We can query tso from PD here.

Reply:
Thanks for your comment, I will correct it.

Reply:
Note that our tso is `milliseconds-since-epoch << 18 + logical-ts`